global.d.ts

```diff
@@ -42,6 +50,9 @@ interface MetadataAndTest<Fn> {
 
     interface Test {
       metadata: MongoDBMetadataUI;
+
+      /** @deprecated Attach spec to a test if you need access to it in a beforeEach hook */
```

Comment:
Just curious - how come this is deprecated?  And what's the intended replacement?

Reply:
Deprecated is the the wrong word, let me leave a comment, we shouldn't need to attach the spec to a test object to get access to it inside a beforeEach I feel like its hack-y, but it's also the way we can get the legacy spec tests to use our new way of skipping tests easily. So I wanted the strikethough to discourage use, a very VSCode solution, but then again maybe this is the way we should do things. Right now the unified runner does everything from within the `it`, if we wanted to refactor it to use hooks we'd need to get the currentTest's spec description to the hook somehow